assets/prototype/application/services/toaster/LoadingToast.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Spinner } from '@blueprintjs/core';
+import { LoadingToastProps } from './types';
+
+const LoadingToastNotice: React.FC<LoadingToastProps> = ({ message }): JSX.Element => {
+	return (
+		<div
+			style={{
+				display: 'flex',
+				flexFlow: 'row nowrap',
+				alignContent: 'flex-start',
+				justifyContent: 'flex-start',
```

Comment:
If this is a temp, then :+1: 
Otherwise it looks a bit weird these camelCase props for css.
Instead we can consider `css` from from `@emotion/core`:
```
import { css, jsx } from '@emotion/core'

const color = 'darkgreen'

render(
  <div
    css={css`
      background-color: hotpink;
      &:hover {
        color: ${color};
      }
    `}
  >
    This has a hotpink background.
  </div>
)
```

`

Reply:
ya we'll be redoing all of the css at some point, likely very soon